web/pgadmin/static/js/helpers/Notifier.jsx

```diff
@@ -68,6 +70,20 @@ AlertContent.propTypes = {
   cancelLabel: PropTypes.string,
 };
 
+function createButtonConfig(icon, label, onOkClick, closeModal, type, color=null, focus=false){
```

Comment:
why do we need such functions which solves nothing?

Reply:
Just to ensure That the code stays DRY for better readability and flexibility.
If want to remove?

Reply:
This is destroying the readability.